### DISCUSSION TITLE
Is mask scaling for image2image working?

### DISCUSSION THREAD

**Discussion by lendrick** [hlky/stable-diffusion-webui#841] [Q&A]
https://github.com/sd-webui/stable-diffusion-webui/pull/715

It looks like this was merged, but I don't see any indication that it's working.  Is there something I need to do?

**Comment by trufty** [comment]
It was only merged into [sd-webui:old-dev](https://github.com/sd-webui/stable-diffusion-webui/tree/old-dev) for now not Master yet

**Comment by codedealer** [comment]
@lendrick it may have been mangled during the merge. Img2img had the most number of conflicts.

**Comment by DasWookie** [comment]
Yeah, the merge has completely borked clean installs it looks like
